### ISSUE TITLE
Berlin Tempelhof Airport

### ISSUE THREAD

**Issue by missinglink** [whosonfirst-data/whosonfirst-data#1826]
This is really minor but I found it today and wanted to log it somewhere:

The following record has many errors: `Berlin Tempelhof Airport`:
https://github.com/whosonfirst-data/whosonfirst-data-admin-xy/blob/master/data/102/548/983/102548983.geojson

- The [airport was closed in 2008](https://en.wikipedia.org/wiki/Berlin_Tempelhof_Airport) and since replaced with [a recreation ground](https://en.wikipedia.org/wiki/Tempelhofer_Feld).
- The polygon is not correct and I'm not sure what it's trying to represent

It might take some work to fix this but at a minimum we should update the `cessation` date to indicate it's not longer a functioning airport, the shape of the old airport is still easily visible in aerial/satellite imagery and should look something [like this](https://www.openstreetmap.org/relation/7317281).

**Comment by stepps00** [comment]
Thanks @missinglink.. those `woedb` geometries aren't always the most accurate. I think you're right in saying the record needs a cessation date.

The easiest thing to do would be to add a cessation date, flag as not current, then supersede it into a new record (in the `*-admin-de` repo).

**Comment by thisisaaronland** [comment]
The shape of the airport is derived from the Flickr Alpha Shapes. Absent any other WOF-compatible data source it's what we've got (for airports) and scopes the problem space to "Berlin-ish" rather than entire planet.

Flagging the record as cessated would definitely be good. @stepps00 is this something the `wof-editor` does yet?

And yes, it should be moved in to the `admin-de` repo. It does not need to be superseded, though.

**Comment by stepps00** [comment]
> Flagging the record as cessated would definitely be good. @stepps00 is this something the wof-editor does yet?

Yep, this is something the tool can handle. Here's a PR that I'll merge shortly that simply flags the existing record with a cessation date and `mz:is_current` = 0:

https://github.com/whosonfirst-data/whosonfirst-data-admin-xy/pull/18/files

Let's keep this issue open until the `XY` -> `DE` wrangling is complete.
